src/generated/resources/assets/gtceu/lang/en_us.json

```diff
@@ -1602,14 +1602,12 @@
   "compass.node.gtceu.materials/block": "Block",
   "compass.node.gtceu.materials/bolt": "Bolt",
   "compass.node.gtceu.materials/chainsaw_head": "Chainsaw Head",
-  "compass.node.gtceu.materials/chipped_gem": "Chipped Gem",
```

Comment:
Whoops! You have to enable this config for dev.

Reply:
Sorted Ty